app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java

```diff
@@ -96,6 +100,45 @@ private void onNotFound() {
         }
     }
 
+
+    /**
+     * Helper method that will split events into upcoming / past
+     * and add to the adapter of this list fragment
+     *
+     * @param events
+     *      The events list to be added
+     */
+    private void splitEventsAndAddToAdapter(Collection<? extends SimpleEvent> events) {
+        Pair<List<SimpleEvent>, List<SimpleEvent>> pair = splitEventsList(events);
+        mAdapter.addAll(pair.first);
+        mAdapter.addAll(pair.second);
```

Comment:
Yeah exactly. That's what I said in the original PR when this is first
implemented. Splitting means here that showing the upcoming items first. It
would be better to do like that but there are too many things to do for me.

On Wed, Sep 9, 2015, 6:36 PM Stefan Hoth <notifications@github.com> wrote:

> In
> app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java
> <https://github.com/gdg-x/frisbee/pull/486#discussion_r39058377>:
>
> > @@ -96,6 +100,45 @@ private void onNotFound() {
> >          }
> >      }
> >
> > +
> > +    /**
> > +     * Helper method that will split events into upcoming / past
> > +     * and add to the adapter of this list fragment
> > +     *
> > +     * @param events
> > +     *      The events list to be added
> > +     */
> > +    private void splitEventsAndAddToAdapter(Collection<? extends SimpleEvent> events) {
> > +        Pair<List<SimpleEvent>, List<SimpleEvent>> pair = splitEventsList(events);
> > +        mAdapter.addAll(pair.first);
> > +        mAdapter.addAll(pair.second);
>
> But first it's split and that added to the same adapter without any
> distinction. I'm just wondering of the same could be achieved by a
> comparator.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/486/files#r39058377>.
>